### PR TITLE
Executive 2021 07 02

### PR DESCRIPTION
**Pull request by LongForWisdom**

### CODE REVIEW

File: governance/votes/Executive vote - July 2, 2021.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### July Core Unit Budget Distributions
+
+Due to the positive outcomes of multiple [Core Unit Budget Proposals](https://mips.makerdao.com/mips/list?&search=$AND(%23core-unit,%20%23budget,%20%23active)&subproposalsMode=true), the following budget allocations will be made to fund operations in the month of July if this executive proposal passes.
```
Comment (prose11) [inline governance/votes/Executive vote - July 2, 2021.md:78]: Link comes up with no MIPs in the portal

File: governance/votes/Executive vote - July 2, 2021.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+**155,000 DAI** will be distributed to the **[Real World Finance (RWA-001)](https://mips.makerdao.com/mips/details/MIP39c2SP1)**  core unit at **0x9e1585d9CA64243CE43D42f7dD7333190F66Ca09**  as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP12).
+
+**182,000 DAI** will be distributed to the **[Risk (RISK-001)](https://mips.makerdao.com/mips/details/MIP39c2SP2)**  core unit at **0xd98ef20520048a35EdA9A202137847A62120d2d9**  as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP13).
+
+**273,334 DAI** will be distributed to the **[GovAlpha (GOV-001)](https://mips.makerdao.com/mips/details/MIP40c3SP11)** core unit at **0x01D26f8c5cC009868A4BF66E268c17B057fF7A73**  as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP3). Note that this includes the GovAlpha continuity budget.
```
Comment (prose11) [inline governance/votes/Executive vote - July 2, 2021.md:84]: Links appear to have been mixed up. The first link should be https://mips.makerdao.com/mips/details/MIP39c2SP3

Second link should be https://mips.makerdao.com/mips/details/MIP40c3SP11

File: governance/votes/Executive vote - July 2, 2021.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+**126,117 DAI** will be distributed to the **[Growth (GRO-001)](https://mips.makerdao.com/mips/details/MIP39c2SP4)**  core unit at **0x7800C137A645c07132886539217ce192b9F0528e**  as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP4).
+
+**44,375 DAI** will be distributed to the **[Content Production (MKT-001)](https://mips.makerdao.com/mips/details/MIP39c2SP5)**  core unit at **0xDCAF2C84e1154c8DdD3203880e5db965bfF09B60** as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP41c4SP5) and modified within this forum [post](https://forum.makerdao.com/t/adding-content-production-multisig/8428/5).
+
+**419,677 DAI** will be distributed to the **[Oracles (ORA-001)](https://mips.makerdao.com/mips/details/MIP39c2SP13)** core unit operational wallet at **0x2d09B7b95f3F312ba6dDfB77bA6971786c5b50Cf** as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP15).
```
Comment (prose11) [inline governance/votes/Executive vote - July 2, 2021.md:92]: Was confused where this figure came from at first, the budget MIP is a little unclear IMO. Might have Pablo update it later

File: governance/votes/Executive vote - July 2, 2021.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
+---
+title: Template - [Executive Vote] Rates Changes, July Core Unit Budgets, and PSM DC-IAM - July 2, 2021
+summary: Vote for Rates Changes, July Core Unit Budgets, and PSM DC-IAM
+date: 2021-07-02T00:00:00.000Z
```
Comment (prose11) [inline governance/votes/Executive vote - July 2, 2021.md:4]: Should this be 16:00 when they are scheduled to go up?

File: governance/votes/Executive vote - July 2, 2021.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+**Core Unit Budget Distributions - July**  
+DAI will be distributed to the Core Units as specified within their ratified budget proposals.
+
+| Core Unit | Distribution |
```
Comment (prose11) [inline governance/votes/Executive vote - July 2, 2021.md:23]: I'm assuming SES will be in next weeks

**Review by prose11 — CHANGES_REQUESTED** [review]
Couple minor link fixes needed

File: governance/votes/Executive vote - July 2, 2021.md (hunk 1 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - July 2, 2021.md:78]: Ahhh, it got removed. I guess I'll just get rid of the link.

File: governance/votes/Executive vote - July 2, 2021.md (hunk 3 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - July 2, 2021.md:92]: Nik updated it on the forum already, he said he was going to do a PR for the repo version too.

File: governance/votes/Executive vote - July 2, 2021.md (hunk 5 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - July 2, 2021.md:23]: Next months, apparently they're fine with that.

File: governance/votes/Executive vote - July 2, 2021.md (hunk 4 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - July 2, 2021.md:4]: Mm, we never hit that anyway. I would rather keep it at midnight, I think.

**Review by prose11 — APPROVED** [review]
LGTM!

**Pull request merged**